core/store/src/trie/update.rs

```diff
@@ -33,6 +33,31 @@ impl TrieUpdate {
             self.trie.get(&self.root, key).map(|x| x.map(DBValue::from_vec))
         }
     }
+
+    /// Get values in trie update for a set of keys.
+    // This function will commit changes. Need to be used with caution
```

Comment:
///

Reply:
I use `//` for internal comment so that it does not get into docstring.